libs/entitlement/src/main/java/org/elasticsearch/entitlement/runtime/policy/PolicyManager.java

```diff
@@ -555,11 +555,7 @@ private void notEntitled(String message, Class<?> callerClass, String componentN
             var moduleName = callerClass.getModule().getName();
             var loggerSuffix = "." + componentName + "." + ((moduleName == null) ? ALL_UNNAMED : moduleName);
             var notEntitledLogger = LogManager.getLogger(PolicyManager.class.getName() + loggerSuffix);
-            String frameInfoSuffix = StackWalker.getInstance(RETAIN_CLASS_REFERENCE)
-                .walk(this::findRequestingFrame)
-                .map(frame -> "\n\tat " + frame)
-                .orElse("");
-            notEntitledLogger.warn("Not entitled: " + message + frameInfoSuffix);
+            notEntitledLogger.warn("Not entitled: " + message, exception);
```

Comment:
Maybe use a simpler log message like "Entitlement blocked" or something. Like you said, the message is redundant.

Reply:
Yeah I held off just because our alert is looking for this string, including the colon. 😅 

I can remove the `+ message` part for sure.